scopes/dependencies/dependency-resolver/dependency-linker.ts

```diff
@@ -580,6 +585,9 @@ export class DependencyLinker {
 
     const packageName = `@teambit/${name}`;
     const target = path.join(rootDir, 'node_modules', packageName);
+    if (this._currentBitDir) {
```

Comment:
I wonder if we need to handle this also in some places of `load-bit.ts`
for example we have there `getMainAspect` which later used to link the `@teambit/bit` itself. which I think will now point to the wrong location.
you can also look on `linkBit` in this file, and see if you get the correct location or not in action.

Reply:
yes, you are right. May I just update the getAspectDir to alway prefer reading from currentBitDir? Is there any scenario where currentBitDir should not be used?

Reply:
@zkochan yes you can change the `getAspectDir`